internal/xds/translator/authentication.go

```diff
@@ -219,11 +219,11 @@ func buildClusterFromJwks(jwks *jwksCluster) (*cluster.Cluster, error) {
 				},
 			},
 		},
-		Http2ProtocolOptions: &core.Http2ProtocolOptions{},
```

Comment:
unrelated to this PR, but curious why the auth cluster is HTTP2 @danehans ?

Reply:
@arkodg I was following the rate limit cluster [specs](https://github.com/envoyproxy/gateway/blob/v0.3.0-rc.1/internal/xds/translator/ratelimit.go#L295). If this is unneeded, can you submit an issue and assign me or create a PR to fix so it can be in the v0.3 release?

Reply:
for RL, we know for sure that the endpoint is a gRPC endpoint supporting HTTP2, for this case we cannot be sure, will raise a PR to change it